.github/workflows/05-examples.yaml

```
@@ -75,29 +75,29 @@ jobs:
         working-directory: ./examples/asa
         run: |
           pipenv run yarn algob deploy
-          pipenv run yarn algob run scripts/transfer/gold-contract-sc.js
-          pipenv run yarn algob run scripts/transfer/gold-delegated-lsig.js
-          pipenv run yarn algob run scripts/transfer/gold-to-john.js
-          pipenv run yarn algob run scripts/transfer/master-fund-john.js
-          pipenv run yarn algob run scripts/transfer/tesla-to-john.js
+          pipenv run yarn algob run --script  scripts/transfer/gold-contract-sc.js
```

```suggestion
          pipenv run yarn algob run --script scripts/transfer/gold-contract-sc.js
```
There should be one space only. The same for all `--script  `

---

Yes, done